src/css/profile/mobile/common/core.less

```diff
@@ -134,6 +134,11 @@
 	overflow-y: visible;
 	overflow-x: hidden;
 	flex-shrink: 1;
+	mask-border: url(images/0_Round_corner/round.svg) 77;
```

Comment:
What's mean the "77" ?

Reply:
In the GUI Guide, there are "top_left.svg" ,"top_right.svg","buttom_left.svg" "bottom_right.svg".
Do you use the  svg file?

Reply:
Need to check that I just copy the style related with the mask from the 'ui-content-area'

Reply:
77 is for the "mask-border-slice"
```
<'mask-border-slice'>
The dimensions for slicing the source image into regions. Up to four values may be specified. See mask-border-slice.
```
round.svg is draw the circle(radius 78). So I can change it to the 78.

Reply:
![image](https://user-images.githubusercontent.com/29908741/79933988-6a2f6400-848c-11ea-9a23-cbcc4b896ff4.png)
Oops,,, It is result when changing the slice value to the 78.
77 pixel seems a correct value

Reply:
ok. It's no problem.
Thanks for your checking.